openy_map.info.yml

```diff
@@ -10,3 +10,4 @@ dependencies:
   - openy_data_wrapper:openy_data_wrapper
   - geolocation_google_maps:geolocation_google_maps
   - openy_upgrade_tool:openy_upgrade_tool
+  - y_branch:y_branch
```

Comment:
```suggestion
  - drupal:y_branch
```

Reply:
This module is on drupal.org so less confusing would be suggested code
See https://ds-docs.y.org/docs/development/dependencies-in-drupal-info.yml/

Reply:
Ah yes I remember now this came up in https://github.com/YCloudYUSA/y_lb/issues/56... I still don't really understand how/why this is different for YMCA WS than other Drupal projects, but feel free to make the changes that make sense to you (:

Reply:
@andriokha you should apply suggestion if you agree, I have no writing rights to your fork

Reply:
addressed here https://github.com/open-y-subprojects/openy_map/pull/59

Reply:
Thanks @podarok, sorry didn't consider you couldn't apply the suggestion, wasn't being difficult! Btw I think there were other drupal-hosted modules that were in the format of PROJECT_NAME:MODULE_NAME in the list.